app/models/institution_builder.rb

```diff
@@ -19,6 +19,15 @@ def self.columns_for_update(klass)
     klass::COLS_USED_IN_INSTITUTION.map(&:to_s).map { |col| %("#{col}" = #{table_name}.#{col}) }.join(', ')
   end
 
+  def self.add_vet_tec_provider(version_number)
```

Comment:
Can we add a spec to `spec/models/institution_builder_spec` to test this method?

Reply:
I went ahead and added a test in spec/models/institution_builder_spec where we are checking to see that vet_tec_provider  is being set to true for an institution when it's weams record has a facility code with a 'V' as it's second character.